### PR TITLE
Fix dark mode switcher flash bug

### PR DESCRIPTION
**Pull request by NriotHrreion**
issue: #4167

### CODE REVIEW

File: layouts/partials/html-head.hbs [hunk 1]
```diff
@@ -34,4 +34,6 @@
   {{#each site.feeds}}
   <link rel="alternate" href="/{{ ../site.locale }}/{{ link }}" title="{{ text }}" type="application/rss+xml">
   {{/each}}
+
+  <script src="/static/js/themeSwitcher.js"></script>
```
Comment (U-C-S) [inline layouts/partials/html-head.hbs:38]: will it be a good idea to just embed the JavaScript code into HTML file, instead of a new js file??

Comment (NriotHrreion) [reply layouts/partials/html-head.hbs:38]: I think it's better to be a new js file instead embedding into the html.

**Comment by MaledongGit** [comment]
@U-C-S: Is it fine with you? Can I help to merge it?

**Comment by XhmikosR** [comment]
This results in duplicate code in main.js and the new JS file. Why can't we have everything in a HEAD script?

**Pull request merged**